climada/entity/exposures/litpop/litpop.py

```diff
@@ -0,0 +1,1199 @@
+        """init LitPop exposure object for a list of countries (admin 0).
+        Sets attributes `ref_year`, `tag`, `crs`, `value`, `geometry`, `meta`,
+        `value_unit`, `exponents`,`fin_mode`, `gpw_version`, `reproject_first`,
+        and `admin1_calc`.
+
+        Alias: set_country()
+
+        Parameters
+        ----------
+        countries : list with str or int
+            list containing country identifiers:
+            iso3alpha (e.g. 'JPN'), iso3num (e.g. 92) or name (e.g. 'Togo')
+        res_arcsec : float (optional)
+            Horizontal resolution in arc-sec.
+            The default is 30 arcsec, this corresponds to roughly 1 km.
+        exponents : tuple of two integers, optional
+            Defining power with which lit (nightlights) and pop (gpw) go into LitPop. To get
+            nightlights^3 without population count: (3, 0).
+            To use population count alone: (0, 1).
+            Default: (1, 1)
+        fin_mode : str, optional
+            Socio-economic value to be used as an asset base that is disaggregated
+            to the grid points within the country
+            * 'pc': produced capital (Source: World Bank), incl. manufactured or
+                    built assets such as machinery, equipment, and physical structures
+                    (pc is in constant 2014 USD)
+            * 'pop': population count (source: GPW, same as gridded population)
+            * 'gdp': gross-domestic product (Source: World Bank)
+            * 'income_group': gdp multiplied by country's income group+1
+            * 'nfw': non-financial wealth (Source: Credit Suisse, of households only)
+            * 'tw': total wealth (Source: Credit Suisse, of households only)
+            * 'norm': normalized by country
+            * 'none': LitPop per pixel is returned unchanged
+            The default is 'pc'.
+        total_values : list containing numerics, same length as countries, optional
+            Total values to be disaggregated to grid in each country.
+            The default is None. If None, the total number is extracted from other
+            sources depending on the value of fin_mode.
+        admin1_calc : boolean, optional
+            If True, distribute admin1-level GDP (if available). Default: False
+        reference_year : int, optional
+            Reference year. Default: CONFIG.exposures.def_ref_year.
+        gpw_version : int, optional
+            Version number of GPW population data.
+            The default is GPW_VERSION
+        data_dir : Path, optional
+            redefines path to input data directory. The default is SYSTEM_DIR.
+        reproject_first : boolean, optional
+            First reproject nightlight (Lit) and population (Pop) data to target
+            resolution before combining them as Lit^m * Pop^n?
+            The default is True. Warning: Setting this to False affects the
+            disaggregation results - expert choice only
+
```

Comment:
Good point :) this is an expert level functionality added if someone wants to explore the disaggregation beyond the scope of the LitPop paper. I would suggest to leave it in but also open to completely throw it out. what do you suggest?

Reply:
I think it is best to keep methods lean. If extra expert options are to be offered, better make another method.

Reply:
Do you suggest to make an alternative `_get_litpop_single_polygon` - method for `reproject_first == False`?

Reply:
mmh i'm tempted to say yes

Reply:
After considering a split of the method and that so far there really is no use case for `reproject_first=False` so far and I don't see much interest to explore the differences between the two approaches, I decided to follow your advice to keep the method lean and get rid of `reproject_first`. If someone would want to turn the order of resampling and the core calculation upside down, this can still result in a new method.